examples/scaling-criteo/03-Training-with-Merlin-Models.ipynb

```diff
@@ -0,0 +1,417 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    model.fit(train, validation_data=valid, batch_size=BATCH_SIZE, callbacks=[validation_callback], epochs=EPOCHS, verbose=1)
</pre><p>Please remove callbacks=[validation_callback] here. it is not valid with this way. you can remove <code>verbose</code> param as well. </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/Merlin/pull/380/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='15'/>